python/cuml/cuml/dask/common/utils.py

```diff
@@ -137,12 +136,8 @@ def persist_across_workers(client, objects, workers=None):
     if workers is None:
         workers = client.has_what().keys()  # Default to all workers
 
-    if check_min_dask_version("2020.12.0"):
-        with dask.annotate(workers=set(workers)):
-            return client.persist(objects)
```

Comment:
Why are we able to remove this?

Reply:
The version of dask rapids supports is `2025.2.0`, `dask.annotate` has been around for 4+ years, no need to still support older versions.